### PR TITLE
Greta/filter vault

### PR DESCRIPTION
**Pull request by gretahuang**
Add filtering to vault. Temporarily has an ugly picker on it until we figure out designs.

### CODE REVIEW

File: src/screens/PdfScreen [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
+import React from 'react';
```
Comment (jasonzbao) [inline src/screens/PdfScreen:2]: Could you remove this file please!

**Review by jasonzbao — COMMENTED** [review]
LGTM couldn't test though (need web PR)

File: assets/images/menulist.svg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 18.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+   viewBox="0 0 700 700" enable-background="new 0 0 700 700" xml:space="preserve">
+<polygon opacity="1" fill="#4A4A4A" points="212,283 487.7,283 349.8,422.5 "/>
+</svg>
```
Comment (lbkchen) [inline assets/images/menulist.svg:6]: lol I think we have icons for this btw in @expo/icons

File: src/screens/LoginScreen.js [hunk 3]
```diff
@@ -54,7 +54,7 @@ export default class LoginScreen extends React.Component {
       error: this.state.errors,
       fields: {
         email: {
-          value: 'kenchen@berkeley.edu',
+          value: 'gretahuang@berkeley.edu',
```
Comment (lbkchen) [inline src/screens/LoginScreen.js:57]: 👀

File: src/screens/VaultScreen.js [hunk 4]
```diff
@@ -15,143 +17,80 @@ import { APIRoutes } from '../helpers/routes/routes';
...
-                {this.filterScroller()}
-              </ScrollView>
-              <View style={styles.contentContainer}>
-                <Resource
-                  veteranId={this.props.navigation.state.params.id}
-                  categories={this.state.categories}
-                  categoriesToDisplay={this.categoriesToDisplay()}
+          <View style={ styles.contentContainer }>
+            <List containerStyle={{ borderTopWidth: 0, borderBottomWidth: 0 }}>
+              <FlatList
```
Comment (lbkchen) [inline src/screens/VaultScreen.js:76]: nice nice nice

**Review by lbkchen — COMMENTED** [review]
Add a screenshot/screen capture of your PR next time 🎉

**Pull request merged**